node.gyp

```diff
@@ -947,6 +947,18 @@
           ],
         }],
       ]
+    },
+    {
```

Comment:
Is it possible to put this whole definition in a separate gyp file in test/ somewhere instead of node.gyp, since it's only used for a test?